.travis.yml

```diff
@@ -64,7 +64,7 @@ jobs:
         - stage: Test
           script:
               - docker build --build-arg DMD=2.071.2.s* --build-arg DIST=xenial
-                -t dmqnode -f docker/Dockerfile.dmqnode .
+                -t dmqnode .
```

Comment:
This is usually [not recommended](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#understand-build-context), as you'll copy the whole project directory to the docker container context. For example the whole `.git` DB (and temporary tests files that you might have). We started putting all in the root dir and then moved to a subdirectory.

Reply:
I see what you mean, but what is the difference here. In both cases the build context is the same. Only the location of the Dockerfile changed.
And honestly I don't see how this could be improved, since almost the whole directory structure is needed for building the node. I'm going to add a `.dockerignore` though, to blacklist some files.

Reply:
Also makd seems to needs the `.git ` directory.

Reply:
You are right, sorry for the noise.